src/helm/benchmark/run_specs/audio_run_specs.py

```diff
@@ -90,3 +97,67 @@ def get_covost2_run_spec(source_language: str, target_language: str) -> RunSpec:
         metric_specs=metric_specs,
         groups=["covost2"],
     )
+
+
+@run_spec_function("vocal_sound")
+def get_vocal_sound_run_spec() -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.audio_language.vocal_sound_scenario.VocalSoundScenario",
+    )
+    adapter_spec = _get_generation_adapter_spec(
+        instructions='Listen to the audio and classify the speaker behavior. Choose only from these options: \
+            "Cough", "Laughter", "Sigh", "Sneeze", "Sniff", or "Throatclearing". Respond with just the behavior.',
```

Comment:
should it be "Throat clearing"? If this is what is in the orignal dataset, can we update the correct Reference answer text to be "Throat clearing".

Reply:
You are right, fixed this